lib/core/test/unit/Cardano/Wallet/Primitive/ModelSpec.hs

```diff
@@ -496,6 +503,8 @@ prop_changeUTxO_inner pendingTxs =
           -- No outputs are omitted when we select everything:
         , UTxO.size utxoAll == F.sum (F.length . view #outputs <$> pendingTxs)
         ]
+    & counterexample ("size utxoAll = " <> show (UTxO.size utxoAll))
+    & counterexample ("sum outputs = " <> show (F.sum (F.length . view #outputs <$> pendingTxs)))
```

Comment:
Request: try to fit this within the line length limit.

Suggestion: use the `report` combinator, as this handles pretty-printing for you (no need to call `show`). (Currently, `report` is in `Cardano.Wallet.Primitive.Migration.SelectionSpec`, but this should arguably be moved to `Test.QuickCheck.Extra`!)

```suggestion
    & report                                                                                                                                                                        
        (UTxO.size utxoAll)                                                                                                                                                         
        "UTxO.size utxoAll"                                                                                                                                                         
    & report                                                                                                                                                                        
        (F.sum (F.length . view #outputs <$> pendingTxs))                                                                                                                           
        "F.sum (F.length . view #outputs <$> pendingTxs)"    
```

Reply:
PR https://github.com/input-output-hk/cardano-wallet/pull/2876 moves `report` to a common location, making it easy to reuse here.

Reply:
1105933ed6bcdd7d26ca9fb5d4ea4e7e7b143459